opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
```

Comment:
From https://github.com/opentracing/opentracing-java/pull/7#issuecomment-173808490
> - creates a child Span from a parent Span when the Tracer is not accessible (c.f. your removal of `Span.startChild`),

I do not understand how it is possible that a Tracer is not accessible, nor the point of creating a Span in any environment when no Tracer exists.